sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLLibraryDAO.java

```diff
@@ -127,6 +127,16 @@
 
   public static final String LIBRARY_SELECT_BY_BOX_POSITION_ID = LIBRARIES_SELECT + " WHERE l.boxPositionId = ?";
 
+  private static final String LIBRARY_SELECT_BY_ADJACENT = LIBRARIES_SELECT + " JOIN Sample ON Library.sample_sampleId = Sample.sampleId, "
+      + "Library TargetLibrary JOIN Sample TargetSample ON TargetLibrary.sample_sampleId = TargetSample.sampleId WHERE TargetLibrary.libraryId = ? "
```

Comment:
I'm getting an error in the editLibrary page: `nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'Library.libraryId' in 'where clause'`